src/Tabs/Tabs.test.js

```diff
@@ -1,101 +1,56 @@
 import { mount } from 'enzyme';
```

Comment:
Now that these are separate components, we should have unit tests for each individually.